# -*- coding:utf-8 -*-

from saklient.errors.httpserviceunavailableexception import HttpServiceUnavailableException
from saklient.errors.httpserviceunavailableexception import HttpServiceUnavailableException

# module saklient.cloud.errors.serveroperationfailureexception

class ServerOperationFailureException(HttpServiceUnavailableException):
    ## サービスが利用できません。サーバの操作に失敗しました。
    
    # (class field) default_message = "サービスが利用できません。サーバの操作に失敗しました。"
    
    pass
